roles/storage-cinder/defaults/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+action: provision
+namespace: openstack
+service_account: cinder
+privileged_service_account: cinder-privileged
+
+storage_cinder_template_dir: "{{ role_path }}/templates"
```

Comment:
I think this is not necessary as ansible already looks in this directory for templates.